.github/workflows/ci.yml

```diff
@@ -58,6 +58,30 @@ jobs:
       - name: test
         run: |
           make test
+  linux-gcc48:
+    runs-on: ubuntu-latest
+    container:
+      image: ubuntu:14.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
```

Comment:
you need to specify recursive here, see other jobs

Reply:
updated